hw/Makefile

```diff
@@ -19,6 +19,7 @@ IPS ?= aes           \
        csrng         \
        edn           \
        entropy_src   \
+       dma           \
```

Comment:
Suggest we keep this list sorted

Reply:
Ups, that was a bit late last night. Fixed the off-by-one error.